metricbeat/docs/fields.asciidoc

```diff
@@ -3193,7 +3193,7 @@ The hard limit on the number of file descriptors opened by the process. The hard
 [float]
 == cgroup Fields
 
-Metrics and limits from the cgroup of which the task is a member. cgroup metrics are reported when the process has membership in a non-root cgroup. These metrics are only available on Linux.
+EXPERIMENTAL: Metrics and limits from the cgroup of which the task is a member. cgroup metrics are reported when the process has membership in a non-root cgroup. These metrics are only available on Linux.
```

Comment:
That is the way it looks now with `experimental[]` right now. The warning box shows up when hovering. I was hoping for some more warning colors, but that works.
![screen shot 2016-08-18 at 08 17 04](https://cloud.githubusercontent.com/assets/244900/17763928/389ba1b2-651c-11e6-9194-9e312f4100e5.png)